src/web/app/pages-admin/admin-notifications-page/notifications-table/notifications-table.component.spec.ts

```diff
@@ -27,4 +31,36 @@ describe('NotificationsTableComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with default fields', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  const notificationTableRowModel1: NotificationsTableRowModel = {
```

Comment:
Maybe can move these two constants above all the testcases?

Reply:
Okay!